packages/itwin/imodel-react-hooks/src/FeatureOverrides/useFeatureOverrides.tsx

```diff
@@ -113,23 +108,25 @@ export const FeatureOverrideReactProvider = ({
   );
 
   useEffect(() => {
-    const attach = () =>
-      IModelApp.viewManager.forEachViewport((vp) => {
+    const attach = () => {
+      for (const vp of IModelApp.viewManager) {
         if (!viewFilter || viewFilter(vp)) {
-          vp.featureOverrideProvider = impl;
+          // to do
+          // vp.featureOverrideProviders = impl;
```

Comment:
need to fix this,

Reply:
think youll need to do this
vp.addFeatureOverrideProvider

Reply:
thanks!